cypress/support/pages/home.pageObject.js

```diff
@@ -8,8 +8,15 @@ class HomePageObject extends PageObject {
   }
 
   assertHeaderContainUsername(username) {
-    this.usernameLink
-      .should('contain', username);
+    this.usernameLink.should('contain', username);
+  }
+
+  clickNewArticle() {
+    cy.get('a.nav-link').contains('New Article').click();
```

Comment:
Hard coded selectors in the clickNewArticle function should be avoided. This could lead to brittle tests that are hard to maintain. Consider using data-qa attributes.

Reply:
The same situation as here 
https://mate-parttime.slack.com/archives/C02J1F3EUDB/p1691237811581409

Reply:
The same situation here
https://mate-parttime.slack.com/archives/C02J1F3EUDB/p1691237811581409